mobile/src/styles/AppStylesheet.ts

```diff
@@ -79,4 +79,39 @@ export const AppStylesheet = StyleSheet.create({
     fontSize: 32,
     fontWeight: "700",
   },
+
```

Comment:
If we are only using these for the button popups let's move these into a separate stylesheet file. You can call it something like `PopupStylesheet.ts`. We don't need to have every style within one file.